.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,5 +1,13 @@
 eclipse.preferences.version=1
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.5
-org.eclipse.jdt.core.compiler.compliance=1.5
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
```

Comment:
I just want to confirm, did you intend to have the eclipse settings included in this PR, we don't include eclipse settings in our other projects

Reply:
It looks like this repository was already including the Eclipse settings before now. I'll see if I can configure the git-ignore settings accordingly to exclude it.